### DISCUSSION TITLE
오픈채팅방 가입

### DISCUSSION THREAD

**Discussion by wntown** [storycraft/node-kakao#540] [Q&A] [answered]
안녕하세요 요즘 부쩍 이 모듈에 관심이 많아져 개발중에 있습니다 ㅎㅎ
오픈톡 입장을 하려면
프로필이 3개중 1개 (실제프로필, 오픈톡프로필, 오픈톡1:1프로필)
이렇게 3가지 중 있어야하는데
프로필 설정방법, 오픈톡방 입장시 입장가능 프로필 확인 기능등이 있는거 같습니다

오픈톡 입장시 프로필 제한을 확인하는것은 못찾은 상태이고

const kakao = require("node-kakao");
const CLIENT = new kakao.TalkClient();
CLIENT.channelList.open.joinChannel(link, profile, passcode)

해당 함수를 통해 방 가입하는것은 찾았으나
link 의 경우 String 형식이 아닌거같고.. profile 은

CLIENT.channelList.open.createOpenDirectProfile(template, profile);
CLIENT.channelList.open.createOpenProfile(template)

코드를 통해 생성 가능한거 같습니다만
각 인자값들을 어떤식으로 넣어야 할지 모르겠습니다 ㅜㅜ

해보신 분이 있다면 조언 부탁드립니다!!

[a comment by an automated account is not shown]

**Reply by wntown** [reply]
감사하게도 해결 됐습니다! ㅎㅎ
알려주신 내용 토대로 닉네임으로도 가입할수 있게됐습니다

**Comment by raravel** [comment]
#344

**Reply by wntown** [reply]
어저께 이거보고도 해결이 안되서 질문 올렸었습니다 ㅜㅜ
